runtime/testapps/src/main/resources/com/google/apphosting/runtime/jetty9/senderrorapp/jetty94/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ Copyright 2021 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <runtime>java21</runtime>
+  <application>senderror</application>
+  <version>1</version>
+  <system-properties>
+    <property name="appengine.use.EE8" value="false"/>
+    <property name="appengine.use.EE10" value="false"/>
```

Comment:
@lachlan-roberts @ludoch is this a combination which exists now? i.e. Java21 and Jetty9.4? I thought we default to using Jetty 12 (either EE8 or EE10) for Java21 customers: https://github.com/GoogleCloudPlatform/appengine-java-standard/blob/78bbabee338ba6e4a5701f6a41f81f6a217c6661/appengine_init/src/main/java/com/google/appengine/init/AppEngineWebXmlInitialParse.java#L97-L101

Reply:
@maigovannon this is a combination which exists currently, but they must specifically configure these flags to use Jetty 9.4 on the Java 21 runtime. I believe it defaults to EE10 if none are set.